test/index.js

```diff
@@ -17,70 +13,351 @@ test('returns false for non-functions', function (t) {
 		[],
 		/a/g,
 		'string',
+		'() => {}',
+		'function () {}',
 		42,
-		new Date()
+		new Date(0),
```

Comment:
the 0 isn’t necessary and actually produces a different value; why this change?

Reply:
Just going by the general rule that tests should be deterministic. Not married to this change as it makes no substantive difference either way though